Firmware/Tupfile.lua

```diff
@@ -104,7 +109,8 @@ LDFLAGS += '-Wl,--undefined=uxTopUsedPriority'
 
 
 -- common flags for ASM, C and C++
-OPT += '-Og'
+-- OPT += '-Og'
+-- OPT += '-O2'
 -- OPT += '-O0'
```

Comment:
Leaving these comments here is a bit redundant.
Maybe swap "debug build" and "linker flags" so the writing to OPT is easier to overview.